test/definitions.py

```diff
@@ -11,6 +11,7 @@
 TEST_DIRECTORY = "test"
 UI_DIRECTORY = "ui"
 MANIFEST_FILENAME = "manifest.txt"
+NODE_COMMAND = "node --experimental-specifier-resolution=node --loader ts-node/esm".split()
```

Comment:
No, this flag has nothing to do with syntax `import` or `require`, it only tell node to auto-add `.js` extension when importing or requiring a new module. So we can `import { something } from './file'`. And ts-node will try to add `.ts` extension, too. In this way we can import a js file or ts file and it will work as expected. We can't import a file with `.ts` extension, node doesn't support that.